src/ucp/api/ucp.h

```diff
@@ -538,6 +545,48 @@ typedef enum {
 } ucp_op_attr_t;
 
 
+/**
+ * @ingroup UCP_COMM
+ * @brief UCP AM receive data parameter fields and flags
+ *
+ * The enumeration allows specifying which fields in @ref ucp_am_recv_param_t
+ * are present and receive operation flags are used. It is used to enable
+ * backward compatibility support.
+ */
+typedef enum {
+    UCP_AM_RECV_ATTR_FIELD_REPLY_EP    = UCS_BIT(0),  /**< reply_ep field */
+
+    /**
+     * Indicates that the data provided in @ref ucp_am_data_callback_t callback
+     * can be held by UCP. If UCS_INPROGRESS is returned from the callback,
```

Comment:
i think it should be "can be held by user callback" ?

Reply:
not sure, because it can be held by the user event outside of this callback. Maybe:
"can be held by the user until released using @ref ucp_am_data_relase"?

Reply:
yes